### PR TITLE
[cker] Introduce Add gradient kernel

### PR DESCRIPTION
**Pull request by Aeren1564**
This commit introduces Add gradient kernel.

From draft https://github.com/Samsung/ONE/pull/12417

### CODE REVIEW

**Comment by Aeren1564** [comment]
This implementation has been verified on https://github.com/Samsung/ONE/pull/12417#issuecomment-1911270045.

**Comment by jyoungyun** [comment]
Please add test code together. :)

**Comment by Aeren1564** [comment]
> Please add test code together. :)

Unittest for Add has been added.

File: compute/cker/src/train/BinaryArithmetic.test.cc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (jyoungyun) [inline compute/cker/src/train/BinaryArithmetic.test.cc:2]:
```suggestion
 * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
```

**Review by jyoungyun — APPROVED** [review]
LGTM :)

File: compute/cker/src/train/BinaryArithmetic.test.cc (hunk 1 above)
Comment (hseok-oh) [reply compute/cker/src/train/BinaryArithmetic.test.cc:2]: @Aeren1564 Please update this later

**Pull request merged**